packages/react-art/src/ReactARTHostConfig.js

```diff
@@ -427,6 +427,14 @@ export function unhideTextInstance(textInstance, text): void {
   // Noop
 }
 
+export function clearContainer(container: Container): void {
+  // TODO This doesn't work for anything other than SVG.
+  // Is that okay?
```

Comment:
Would be interested to hear @sebmarkbage's thoughts on the priority of this.

Reply:
Can we also try `container.textContent = ''` as per my other comment? I think it works for SVG too.

Reply:
I'm going to just put a "TODO" here. I just noticed that my Flow types are meaningless anyway b'c the file is missing a @flow pragma and there is no `Container` type for ART (which supports SVG, Canvas, and WebGL).

Reply:
Makes sense :)